tools/Makefile

```diff
@@ -22,7 +22,6 @@ clean distclean:
 #---------------------------------------------------------------------
 
 SET_VERSION_SCRIPTS = \
-	bin/gpactivatestandby \
 	bin/gpaddmirrors \
```

Comment:
Why leave gpaddmirrors?  HAWQ doesn't have mirrors to begin with (legacy from Greenplum database).